### PR TITLE
Long embedding names that break modal for zero-shot fixed

### LINKED ISSUE #6: Long embeddings names (Huggingface models) can break the modal on heuristics
Bildschirmfoto 2022-07-21 um 16 22 05

div with modal-box should include something like
```css
max-width: 80%; 
min-width: 55%;
width:fit-content;
```

### PR DESCRIPTION
**Pull request by lumburovskalina**
Fixed long embedding names that break the modal for zero-shot

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -114,6 +114,10 @@ module.exports = {
       "xxs": "9rem",
       "xs-1": "250px",
       ...defaultTheme.maxWidth,
+      "3/5": "80%"
+    },
+    minWidth: {
+      "1/2": "50%"
```
Comment (JWittmeyer) [inline tailwind.config.js]: why was this added to the tailwind config? and not e.g. a scss class for the needed logic?

Comment (lumburovskalina) [reply tailwind.config.js]: We can use this property for all modals, so I assumed it will be better if we have it as a 'class' instead of writing it as a style.

**Review by JWittmeyer — APPROVED** [review]
